cli/internal/run/run.go

```diff
@@ -197,7 +197,7 @@ func (c *RunCommand) Run(args []string) int {
 			return 1
 		}
 	}
-	filteredPkgs, isAllPackages, err := scope.ResolvePackages(runOptions.scopeOpts(), scmInstance, ctx, c.Ui, c.Config.Logger)
+	filteredPkgs, isAllPackages, err := scope.ResolvePackages(&runOptions.scopeOpts, c.Config.Cwd.ToStringDuringMigration(), scmInstance, ctx, c.Ui, c.Config.Logger)
```

Comment:
That this is called `cwd` throws me for a loop. I want to split "directory in which the command was invoked" from "directory which is being treated as the execution root" and `cwd` is woefully incomplete at expressing that.

Shouldn't be handled in this PR, but as you're thinking about things, this is something to keep in mind.

Reply:
Agreed. I think a rename of the config field is in order at some point.